### DISCUSSION TITLE
And what about Client on LG | WebOS?

### DISCUSSION THREAD

**Discussion by pnsbttl** [jellyfin/jellyfin-desktop#200] [Q&A]
Hello Team,
You're doing an amazing job!
I'm super fan about what you did with Jellyfin, it's working like a charm, Many thanks for that :-)
I enjoy watching all my personal medias from browser and my phone.
I'm wondering when you plan to complete LG TV application.
This is the only missing part in my home media environment.

thank you in advance for your answer!

**Comment by iwalton3** [comment]
There is an LG application but it is unrelated to this one. It apparently is very difficult to get the application approved though so for now it has to be side-loaded.

**Reply by pnsbttl** [reply]
Thank you for your fast answer.
by saying side-loaded, do you mean connecting computer to TV? installing application, on TV, via alternative way than using LG Store?
Thank you in advance for your answer!

**Reply by iwalton3** [reply]
Yes you have to load the application through some kind of developer mode. I have never done it.
